.env.example

```diff
@@ -0,0 +1,3 @@
+# Set this value to true if table data needs to be hidden for retraining
+# Then save this varible or rename this file to .env
+NEXT_PUBLIC_IS_CONSTRUCTION=
```

Comment:
On second thoughts, would it be straight-forward to add this to the `config.ts`? Might be nice to have everything in one place

Reply:
we could but i it would be much more flexible in .env, instead of having to making new pr to change the view you can just adjust the dockerfile or connect the .env to github variables or something